### PR TITLE
Add Modern Robotics 12v motor for FTC

### PR DESCRIPTION
**Pull request by Petelax**
This adds the Modern Robotics motor commonly used in FTC. This motor is often found in the GoBilda Yellow Jacket Planetary Gearbox motors, one of the most popular motors in FTC. This should make your excellent calculators even more useful in FTC. Data is from motors.vex.com.

### CODE REVIEW

**Comment by tervay** [comment]
Deploy preview for _recalc_ ready!

✅ Preview
https://recalc-8tzfsjz5n-tervays-projects.vercel.app

Built with commit 0102c1b4932a39da29cfb96ca63e5127e9e01b87.
This pull request is being automatically deployed with [vercel-action](https://github.com/marketplace/actions/vercel-action)

**Comment by tervay** [comment]
Were you able to test this on calculators to check if the models look OK? For example on the linear calculator. Many brushed motors are currently dysfunctional due to an incorrect inductance model.

**Comment by Petelax** [comment]
I was mainly working with the drivetrain calculator where it looked pretty accurate and the drive feedforward gains worked well on hardware.

The flywheel and intake calculators seems mostly right, the arm calculator seems a little optimistic compared to JVN, particularly with stall load, but is mostly in the same realm. 

I have noticed that most of the linear calculator is non functional on all of the motors we use in FTC (Rev HD Hex and NeveRest), although the stall loads look correct and I can test the feedforward gains on hardware later this week. I might be able to take a look at the actual model to see why the inductance model is bad, but my math isn't the greatest. 

TL;DR: The Modern Robotics motor isn't any more buggy than the Rev and AndyMark equivalents in each calculator.

Also, I noticed you have the Vex V5 smart motor available which doesn't run on 12V. If the math supports motors that run at different voltages, adding some 6V servos used in FTC would be very helpful. Servo are much more common in FTC than in FRC because of the limit on the number of motors per robot. Vex has not tested any of these servos however and I haven't seen any independent measurements of their performance, so I'm hesitant on if you want to deal with that.

**Comment by tervay** [comment]
Thanks for the contribution, sorry for the delay @Petelax .
